### DISCUSSION TITLE
Page Bundles

### DISCUSSION THREAD

**Discussion by lukemalcolm** [facebook/docusaurus#5696] [Q&A]
I've been scouring the documentation to understand if you can bundle the assets for a doc in a subdirectory next the the .md file? 

i.e. :
```
website # root directory of your site
├── docs
│   └── docpage.md
│   └── docpage/
        └── imageA.jpg
        └── imageB.jpg
        └── attachment.txt
```

Is it possible to do this and access via Markdown image tags with relative path? (Rather than having to put everything in the static folder)?

We have lots of images to manage and having them as a 'blob' in /static with its own dir structure isn't really sustainable.

**Comment by slorber** [comment]
Yes, you can use file links and images using relative paths like `![](./myImage.png)`, those are automatically converted to `require` calls and go through the Webpack file-loader system
